code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -470,3 +470,21 @@ obj/item/weapon/storage/bag/plasticbag/quick_store(var/obj/item/I)
 	..()
 	for(var/i = 1 to storage_slots)
 		new /obj/item/weapon/coin/gold(src)
+
+// -----------------------------
+//          Potion Bag
+// -----------------------------
+
+/obj/item/weapon/storage/bag/potion
+	name = "\improper Bag of potions"
+	desc = "Not too dissimilar to the fabled bag of alcohol. The wizard federation is not responsible for possible rainbow puking."
+	icon = 'icons/obj/pbag.dmi'
+	icon_state = "pbag"
+	item_state = "pbag"
+	body_parts_covered = FULL_HEAD|BEARD
+	slot_flags = SLOT_BELT | SLOT_HEAD
+	storage_slots = 50;
```

Comment:
😠

Reply:
Which part in particular makes you angery?
-Using "" instead of '' to specify the item state
-The ";" at the end of the storage number
-The fact that it covers your beard
-Being allowed to put it on your head/belt in the first place
-All of the above

Reply:
@Irradiation 
Code comments always show a handful of lines, but the specific line being referenced by any given code comment is the bottom-most one.
So my guess would be the `;`.

Reply:
:man_shrugging: 
Done.